projects/cdk/services/zoom.service.ts

```diff
@@ -56,6 +55,7 @@ export class TuiZoomService extends Observable<TuiZoom> {
 
                                 return {clientX, clientY, delta, event};
                             }),
+                            takeUntil(typedFromEvent(nativeElement, 'touchend')),
```

Comment:
Let's move `preventDefault` from above to a separate utility operator we have in cdk?

Reply:
Unfortunately, the type of the `preventDefault ` operator does not match the value returned from the stream

Reply:
You just need to move it up before `scan` :)